subscriptions-api.md

```diff
@@ -444,14 +471,12 @@ This filter will select events only with the event type equal to
 
 ```JSON
 {
+  "filters": [{
     "dialect": "basic",
-    "filters": [
```

Comment:
I think this should be `conditions`. So there is just a single filter, but a filter could have multiple `conditions`. So it is not possible to mix dialects but to have multiple `conditions` combined with an implicit logical AND.

Reply:
see above